test/opt/inline_test.cpp

```diff
@@ -1442,33 +1442,33 @@ OpFunctionEnd
       R"(%false = OpConstantFalse %bool
```

Comment:
I changed the test but it only changes the result id because of the id assignment order.